.gitignore

```diff
@@ -242,4 +242,3 @@ launchSettings.json
 /SolutionMappings.json
 /TestMappings.json
 /tools/Modules/tmp
-/tools/Az/Az.psm1
```

Comment:
Don't remove it from .gitignore. See https://github.com/Azure/azure-powershell/issues/7603